docs/bot-usage.md

```diff
@@ -153,10 +154,12 @@ Hyperopt uses an internal json config return by `hyperopt_optimize_conf()`
 located in `freqtrade/optimize/hyperopt_conf.py`.
 
 ```
-usage: freqtrade hyperopt [-h] [-e INT] [--use-mongodb]
+usage: freqtrade hyperopt [-h] [-e INT] [--hyperopt PATH] [--use-mongodb]
```

Comment:
IMO the argument name `--hyperopt` is misleading because the subcommand is already called `hyperopt`